todo_app/templates/index.html

```diff
@@ -4,15 +4,26 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead"></p>ByteOnTheRocks to-do app: So much to do, wake up!</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{item}}</li>
```

Comment:
You could render these items more elegantly. You could but don't have to do it now. You will at one point.

`{{ c.id }}` will give you the id value in HTML, for example and you can build slightly more readable, slick UI if you tinker with HTML and CSS a little bit.

![image](https://user-images.githubusercontent.com/92427951/207849971-8852c964-75b1-4472-920d-af86fdaecf7b.png)